querybook/server/const/metastore.py

```diff
@@ -4,3 +4,32 @@
 class DataTableWarningSeverity(Enum):
     WARNING = 0
     ERROR = 1
+
+
+class MetadataType(Enum):
+    TABLE_DESCRIPTION = "table_description"
+    COLUMN_DESCRIPTION = "column_description"
+    OWNER = "owner"
+    TAG = "tag"
+    DOMAIN = "domain"
```

Comment:
lets remove DOMAIN in this PR for now, since this metadata entity does not actually exist.
Once we do treat domain as something in querybook, we can add it back in